README.rst

```diff
@@ -0,0 +1,134 @@
+   :target: https://travis-ci.org/diffpy/diffpy.pdfmorph
+
+.. image:: http://codecov.io/github/diffpy/diffpy.pdfmorph/coverage.svg?branch=master
+   :target: http://codecov.io/github/diffpy/diffpy.pdfmorph?branch=master
+
+
+PDFmorph
+========================================================================
+
+PDFmorph is a Python software package designed to increase the insight
+researchers can obtain from measured atomic pair distribution functions (PDFs) 
+in a model independent way. The program was designed to help a researcher 
+answer the question: "has my material undergone a phase transition between 
+these two measurements?".
+
+One approach is to compare the two PDFs in a plot and view the difference curve
+underneath. However, significant signal can be seen in the difference curve from
+benign effects such as thermal expansion (peak shifts) and increased thermal 
+motion (peak broadening) or a change in scale due to differences in incident flux,
+for example. PDFmorph will do its best to correct for these benign effects before 
+computing and plotting the difference curve. One measured PDF (typically the higher
+temperature one) is identified as the target PDF and the second PDF is then "morphed"
+by "stretching" (chaning the r-axis to simulate a uniform lattice expansion),
+"smearing" (broadening peaks through a uniform convolution to simulate increased
+thermal motion), and "scaling" (self-explanatory). PDFmorph will vary the amplitude
+of the morphing transformations to obtain the best fit between the morphed and the
+target PDFs, then plot them on top of each other with the difference plotted below.
+
+There are also a few other morphing transformations in the program.
+
+Finally, we note that PDFmorph should work on other spectra that are not PDFs,
+though it has not been extensively tested beyond the PDF.
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, https://github.com/diffpy/diffpy.utils
+
+It is recommended that you use `Anaconda Python <https://www.anaconda.com/distribution/>`_ to conveniently install PDFmorph and its software dependencies 
+with a few concise commands. For example, on Ubuntu Linux some of the required
+software can be installed using ::
+
+	sudo apt-get install \
+	python-setuptools python-wxtools python-numpy \ 
+	python-matplotlib
+
+To install the remaining packages, see installation instructions at their
+respective web-pages.
+
+INSTALLATION
+------------------------------------------------------------------------
+
+PDFmorph is an application which makes use of the Command Line Interface to
```

Comment:
This is not an installation issue so please remove this paragraph from here.  I am not sure we need it in the readme tbh.

Reply:
I moved it up into requirements for now. As far as its necessity, I think it would be useful if people had no idea about how to use a terminal! Would you recommend any particular resource for people to learn from in the docs (maybe the Effective Computing in Physics textbook ch 1?)?